Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:latest
+
+WORKDIR .
+COPY requirements-test.txt ./
+COPY setup.py ./
+RUN apt-get update && apt-get -y install openssl sqlite3 libsqlite3-dev libssl-dev gdal-bin libproj-dev libgeos-dev libspatialite-dev python-pip
```

Comment:
is it possible to wrap this command on multiple lines (2 or 3) in order to improve its readability like this [Dockerfile in django-x509](https://github.com/openwisp/django-x509/blob/master/Dockerfile#L4-L6)?